src/map_ref.rs

```diff
@@ -149,13 +149,15 @@ where
     S: BuildHasher,
 {
     /// Maps `key` to `value` in this table.
+    ///
     /// See also [`HashMap::insert`].
     pub fn insert(&self, key: K, value: V) -> Option<&'_ V> {
         self.map.insert(key, value, &self.guard)
     }
 
-    /// Maps `key` to `value` in this table if, and only if, `key` is not
-    /// already mapped to a value.
+    /// Maps `key` to `value` in this table unless `key` is already mapped to a
+    /// value.
```

Comment:
Any particular reason why this has a different first sentence from `HashMap::try_insert`?

Reply:
> Any particular reason why this has a different first sentence from `HashMap::try_insert`?

I was trying to base it on the first sentence from `HashMapRef::insert`, but we should probably update them both.